packages/google_maps_flutter/google_maps_flutter_web/example/lib/main.dart

```diff
@@ -20,6 +20,9 @@ class MyApp extends StatefulWidget {
 class _MyAppState extends State<MyApp> {
   @override
   Widget build(BuildContext context) {
-    return const Text('Testing... Look at the console output for results!');
+    return const Directionality(
```

Comment:
I accidentally ran the example app once and it did throw an error about a missing Directionality widget

Reply:
This is fine, the skeleton app is only used to have a Flutter context where things run, but IIRC this particular test does not render a map or anything.